### PR TITLE
Focus on current bounding-box when resetting camera-eye on a 3D space view (double click it)

### LINKED ISSUE #5204: Too large accumulated bounding box often leads to poor default camera view
**Describe the annoyance**
A good example is the nuScenes one, where initially some data is logged close to the origin until the first transform for the car is logged; the subsequent data is then far away from the origin, and the accumulated bounding box will be way too big. 

The too large bounding box makes double click behave quite unintuitively as it doesn't focus on what's currently in view, but rather resets the view to something pretty useless.

In my experience this is quite a common issue, that sometimes can be worked around by logging data in a specific order. However, that is not a nice solution. 

**Expected behavior**
As discussed with @Wumpf it might be better to always focus on the current bounding box until the user has moved the camera. 

Maybe adding an option to the Camera section for the 3D blueprint to allow switching between focusing on the accumulated and the current bounding box might be good to not make things worse in cases where the accumulated bounding box is more useful.

Related to https://github.com/rerun-io/rerun/issues/810.

### PR DESCRIPTION
**Pull request by Wumpf**
### What

* Fixes #5204

Bit back and forth on this: Originally also wanted to track the non-accumulated bounding box when data comes in, but for many scenes this gets too awkward.

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5209/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5209/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5209/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5209)
- [Docs preview](https://rerun.io/preview/9a4526d29f61a77f2d9c1e87d61bb86b38392dff/docs) 
- [Examples preview](https://rerun.io/preview/9a4526d29f61a77f2d9c1e87d61bb86b38392dff/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: crates/re_space_view_spatial/src/ui_3d.rs [hunk 1]
```diff
@@ -92,10 +92,12 @@ fn ease_out(t: f32) -> f32 {
 impl View3DState {
     pub fn reset_camera(
         &mut self,
-        scene_bbox_accum: &BoundingBox,
+        scene_bbox: &BoundingBox,
         scene_view_coordinates: Option<ViewCoordinates>,
     ) {
-        self.interpolate_to_orbit_eye(default_eye(scene_bbox_accum, scene_view_coordinates));
+        // Mark as interaction since we want to stop doing any automatic interpolation even after a
```
Comment (teh-cmc) [inline crates/re_space_view_spatial/src/ui_3d.rs:98]: after a what???! :eyes:

**Review by teh-cmc — APPROVED** [review]
Yeah makes more sense I think

File: crates/re_space_view_spatial/src/ui_3d.rs [hunk 2]
```diff
@@ -546,7 +548,7 @@ pub fn view_3d(
                 if space_view_id == &query.space_view_id {
                     state
                         .state_3d
-                        .reset_camera(&state.bounding_boxes.accumulated, scene_view_coordinates);
+                        .reset_camera(&state.bounding_boxes.current, scene_view_coordinates);
```
Comment (emilk) [inline crates/re_space_view_spatial/src/ui_3d.rs:551]: What about the other place `reset_camera` is called from?

Maybe it's better to pass in the full `SceneBoundingBoxes` to `reset_camera`?

Comment (Wumpf) [reply crates/re_space_view_spatial/src/ui_3d.rs:551]: good point actually

File: crates/re_space_view_spatial/src/ui_3d.rs [hunk 3]
```diff
@@ -92,10 +92,13 @@ fn ease_out(t: f32) -> f32 {
...
     pub fn reset_camera(
         &mut self,
-        scene_bbox_accum: &BoundingBox,
+        scene_bbox: &SceneBoundingBoxes,
         scene_view_coordinates: Option<ViewCoordinates>,
     ) {
-        self.interpolate_to_orbit_eye(default_eye(scene_bbox_accum, scene_view_coordinates));
+        // Mark as interaction since we want to stop doing any automatic interpolations,
+        // even if this is caused by a full reset.
+        self.last_eye_interaction = Some(Instant::now());
```
Comment (emilk) [inline crates/re_space_view_spatial/src/ui_3d.rs:100]: When tracking an entity (camera), `reset_camera` is called each frame, I believe.

Won't resetting `last_eye_interaction` each frame cause the center of the orbit camera to show up each frame?

Also, what does this have to do with the PR description?

Comment (Wumpf) [reply crates/re_space_view_spatial/src/ui_3d.rs:100]: Originally I also wanted that thing
```
        if self.last_eye_interaction.is_none() {
            self.interpolate_to_orbit_eye(default_eye(
                &bounding_boxes.accumulated,
                scene_view_coordinates,
            ));
        }
```
to use the current bounding box, but in many example scenes this looks too jerky
